nautobot_firewall_models/forms.py

```diff
@@ -577,6 +577,222 @@ class Meta:
         ]
 
 
+# NATPolicy
+
+
+class NATPolicyRuleFilterForm(BootstrapMixin, StatusModelFilterFormMixin, CustomFieldModelFilterFormMixin):
+    """Filter form to filter searches."""
+
+    field_order = ["q", "name"]
+
+    model = models.PolicyRule
+    q = forms.CharField(
+        required=False,
+        label="Search",
+        help_text="Search within Name or Description.",
+    )
+    name = forms.CharField(required=False, label="Name")
+    tag = TagFilterField(models.PolicyRule)
+
+
+class NATPolicyRuleForm(BootstrapMixin, CustomFieldModelFormMixin, RelationshipModelFormMixin):
```

Comment:
I see the mode attribute on the NATPolicyRule model but it's missing from the form. Same with status.

Reply:
If you look at the rendered form on Nautobot, the Status is included. AFAIK you only need to explicitly include fields in a form as class variables if you want do change any of the default behavior. If you check `NATPolicyRuleForm.Meta.fields` `status` is included there.

Reply:
I wasn't aware of that. Good to know.